guiEditor/src/guiEditor.ts

```diff
@@ -57,7 +56,7 @@ export class GUIEditor {
 
         if (options.customLoadObservable) {
             options.customLoadObservable.add((data) => {
-                SerializationTools.Deserialize(data, globalState);
+                //TODO: Add deserilization here.

```

Comment:
are you tracking this somehow?

Reply:
You mean in github?

Reply:
yep, some issue you can reference here, or some external todo list?
TODOs tend to stay in place...

Reply:
Making git issues for next PR.